app/models/components/course/duplicator_component_host.rb

```diff
@@ -0,0 +1,15 @@
+class Duplicator::ModelComponentHost
```

Comment:
Isn't it supposed to be CamelCase? I followed model_component_host.rb.

I thought a new host would be cleaner so stuff like assessments would have to allow duplication. Seems a bit strange for them to be part of a course model host.